index.html

```diff
@@ -0,0 +1,212 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=Edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <script src="https://kit.fontawesome.com/3b9cd09213.js" crossorgin="anonymous"></script>
+    <link rel="preconnect" href="https:// fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link href="https://fonts.googleapis.com/css2?family=poppins:wght@500&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    <a href="https://fonts.googleleapis.com/css2?family=Material+Symbols+Rounded:ops,wght">
+    <title>MyPorfolio</title>
+</head>
+
+<body>
+    <section class="wraper">
+        <header>
+            <nav>
+                <a href="#" class="logo"><img src="./images/My Logo.png"></a>
+                <ul class="list">
+                <li><a href="#">Portfolio</a></li>
+                <li><a href="#">About</a></li>
+                <li><a href="#">Contact</a></li>
+            </ul>
+                <img src="./images/Hamburger.png" alt="Hamburger" class="hamburger">
+            </nav>
+        </header>
+        <section class="headline">
+            <h1 >Hello,I'm Khanya glad to see you! </h1>
+            <p class="details">
+                I'm a software developer! I can help you build a product, feature or
+                website Look through some of my work and experience! If you like what
+                you see and have a project you need coded, don't hesitate to contact me.
+            </p>
+            <h3>LET'S CONNECT</h3>
+            <ul class="social">
+                <li>
+                <a href="#"><img class="social-icons" src="./images/github.png" alt="github"></a>
+                </li>
+                <li>
+                  <a href="#"><img class="social-icons" src="./images/linkedin.png" alt="Linkedin"></a>
+                </li>
+               <li>
+                 <a href="#"><img class="social-icons" src="./images/fingure.png" alt="fingure icon"></a>
+                </li>
+                <li>
+                <a href="#"><img class="social-icons" src="./images/tweet.png" alt="twitter"></a>
+                </li>
+                <li>
+                <a href="#"><img class="social-icons" src="./images/medium.png" alt="medium icon"></a>
+                </li>
+              </ul>
+        </section>
+    </div>
```

Comment:
- [OPTIONAL] Kindly  consider removing this closing div tag as it doesn't have the opening tag